nepalingo-web/src/App.tsx

```diff
@@ -6,22 +6,27 @@ import DictionaryPage from "@/pages/DictionaryPage";
 import { BrowserRouter as Router, Route, Routes } from "react-router-dom";
 import ReactGA from "react-ga4";
 import { PrivateRoutes } from "@/components/PrivateRoutes";
+import { LanguageProvider } from "./contexts/LanguageContext";
+import RandomQuoteComponent from "./components/randomQuotes";
 
 const App: React.FC = () => {
   const TrackingID = import.meta.env.VITE_GOOGLE_ANALYTICS_TRACKING_ID;
   ReactGA.initialize(TrackingID);
 
   return (
-    <Router>
-      <Routes>
-        <Route path="/login" element={<Login />} />
-        <Route element={<PrivateRoutes />}>
-          <Route path="/" element={<Home />} />
-          <Route path="/flashcard" element={<FlashcardPage />} />
-          <Route path="/dictionary" element={<DictionaryPage />} />
-        </Route>
-      </Routes>
-    </Router>
+
+    <><LanguageProvider>
+      <RandomQuoteComponent></RandomQuoteComponent>
```

Comment:
this should be inside of the `Home` component - otherwise, it will show up on every page right?